server/pages/index.js

```diff
@@ -408,13 +354,33 @@ module.exports = function() {
 
 	// Isomorphic routing
 	sections
-		.filter( section => section.isomorphic )
 		.forEach( section => {
-			sectionsModule.require( section.module )( serverRouter( app, setUpRoute, section ) );
+			section.paths.forEach( path => {
+				const pathRegex = utils.pathToRegExp( path );
+
+				app.get( pathRegex, function( req, res, next ) {
+					if ( config.isEnabled( 'code-splitting' ) ) {
+						req.context = Object.assign( {}, req.context, { chunk: section.name } );
+					}
+					next();
+				} );
+
+				if ( ! section.isomorphic ) {
+					if ( section.enableLoggedOut ) {
+						app.get( pathRegex, setUpRoute, serverRender );
```

Comment:
Asking without a fuller context: why `setUpRoute` and not `setUpLoggedOutRoute`?